EurKEY.keylayout

```diff
@@ -1,1767 +1,1767 @@
 <?xml version="1.1" encoding="UTF-8"?>
 <!DOCTYPE keyboard SYSTEM "file://localhost/System/Library/DTDs/KeyboardLayout.dtd">
-<!--Last edited by Ukelele version 3.2.2.166 on 2017-02-16 at 16:36 (GMT+1)-->
-<keyboard group="126" id="-14519" name="EurKEY v1.2" maxout="1">
-	<layouts>
-		<layout first="0" last="17" mapSet="16c" modifiers="f4"/>
-		<layout first="18" last="18" mapSet="994" modifiers="f4"/>
-		<layout first="21" last="23" mapSet="994" modifiers="f4"/>
-		<layout first="30" last="30" mapSet="994" modifiers="f4"/>
-		<layout first="194" last="194" mapSet="994" modifiers="f4"/>
-		<layout first="197" last="197" mapSet="994" modifiers="f4"/>
-		<layout first="200" last="201" mapSet="994" modifiers="f4"/>
-		<layout first="206" last="207" mapSet="994" modifiers="f4"/>
-	</layouts>
-	<modifierMap id="f4" defaultIndex="7">
-		<keyMapSelect mapIndex="0">
-			<modifier keys="command?"/>
-			<modifier keys="anyShift? caps? command"/>
-		</keyMapSelect>
-		<keyMapSelect mapIndex="1">
-			<modifier keys="anyShift caps?"/>
-		</keyMapSelect>
-		<keyMapSelect mapIndex="2">
-			<modifier keys="caps"/>
-		</keyMapSelect>
-		<keyMapSelect mapIndex="3">
-			<modifier keys="anyOption"/>
-		</keyMapSelect>
-		<keyMapSelect mapIndex="4">
-			<modifier keys="anyShift caps? anyOption command?"/>
-		</keyMapSelect>
-		<keyMapSelect mapIndex="5">
-			<modifier keys="caps anyOption"/>
-		</keyMapSelect>
-		<keyMapSelect mapIndex="6">
-			<modifier keys="caps? anyOption command"/>
-		</keyMapSelect>
-		<keyMapSelect mapIndex="7">
-			<modifier keys="anyShift caps? option? command? control"/>
-			<modifier keys="shift? caps? anyOption command? control"/>
-			<modifier keys="caps? anyOption? command? control"/>
-		</keyMapSelect>
-	</modifierMap>
-	<keyMapSet id="16c">
-		<keyMap index="0">
-			<key code="0" action="a"/>
-			<key code="1" action="s"/>
-			<key code="2" action="d"/>
-			<key code="3" action="f"/>
-			<key code="4" action="h"/>
-			<key code="5" action="g"/>
-			<key code="6" action="z"/>
-			<key code="7" action="x"/>
-			<key code="8" action="c"/>
-			<key code="9" action="v"/>
-			<key code="10" output="§"/>
```

Comment:
This change seems to have broken the layout on my international keyboard. Ukulele shows a capital ß for my backtick key because of that change. Maybe your find & replace of the § symbol was too aggressive ?

Reply:
What exactly do you mean by "backtick key"? This one: `?

Reply:
@alexmick bump

Reply:
Yes the key next to the `z` key on a QWERTY UK layout : 
![uk](https://upload.wikimedia.org/wikipedia/commons/thumb/0/06/KB_United_Kingdom_Mac_-_Apple_Keyboard_%28MC184B%29.svg/420px-KB_United_Kingdom_Mac_-_Apple_Keyboard_%28MC184B%29.svg.png)

Reply:
Interesting. My backtick key works perfectly fine. The capital ß is on Option+Shift+S. It shows as ẞ for me in case of no support. Not sure how to go about it form here. I testet it on my MacBook Pro and on my desktop keyboard.